examples/feedback/metrics-server/README.md

```diff
@@ -0,0 +1,350 @@
+# Stateful Model Feedback Metrics Server
+In this example we will add statistical performance metrics capabilities by levering the Seldon metrics server.
+
+Dependencies
+* Seldon Core installed
+* KNative eventing v0.11.0
+* KNative serving v0.11.1 (optional)
+
+
+
+```python
+!mkdir -p config
+```
+
+### Create a simple model
+We create a multiclass classification model - iris classifier.
+
+The iris classifier takes an input array, and returns the prediction of the 4 classes.
+
+The prediction can be done as numeric or as a probability array.
+
+
+```bash
+%%bash
+kubectl apply -f - << END
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: multiclass-model
+spec:
+  predictors:
+  - graph:
+      children: []
+      implementation: SKLEARN_SERVER
+      modelUri: gs://seldon-models/sklearn/iris
+      name: classifier
+      logger:
+        mode: all
+    name: default
+    replicas: 1
+END
+```
+
+    seldondeployment.machinelearning.seldon.io/multiclass-model configured
+
+
+#### Send test request
+
+
+```python
+!kubectl run --quiet=true -it --rm curl --image=radial/busyboxplus:curl --restart=Never -- \
+    curl -X POST -v "http://multiclass-model-default.default.svc.cluster.local:8000/api/v1.0/predictions" \
+        -H "Content-Type: application/json" -d '{"data": { "ndarray": [[1,2,3,4]]}, "meta": { "puid": "hello" }}'
+```
+
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+
+
+### Create Metrics Service
+
+
+```python
+%%writefile config/multiclass-service.yaml
```

Comment:
maybe naming file `multiclass-kservice.yaml` (service -> kservice) will be clearer to distinguish that this is knative service and not standard k8s service